htdocs/install/mysql/tables/llx_societe.sql

```diff
@@ -116,6 +116,8 @@ create table llx_societe
   fk_user_creat            integer NULL,                        		-- utilisateur qui a cree l'info
   fk_user_modif            integer,                             		-- utilisateur qui a modifie l'info
 
+  fk_establishment		   integer,
```

Comment:
Why adding this field here ?
The establishment is the table to store the different location of a company where users can works.
So a user has a link to an establishment but why this field here ? Can you provide an example ?

Reply:
A establishment may belong to another establishment?
Ex. A hotel chain

Reply:
@eldy
Yes after reflection maybe I was confused but the goal is to achieve is almost the same.
There are two cases:
- First case: I am the owner of a company, I have 4 sectors and I want to organize my staff so we have to use hrm/establishment/card.php and in this case it's missing fk_establishment_parent in the table llx_establishment (if not is entity? but we do not have the hand to specify it)
- Now listed suppliers/customers, a company can belong to a group, so maybe we need to talk about fk_parent instead of fk_establishment (we do not have where we can specify it). 

This is important in sorting products and knowing the turnover by group and by entity.

Thx